ros2_control/doc/index.rst

```diff
@@ -0,0 +1,32 @@
+Welcome to the documentation for ros2_control
+================================================
+
+For more information of the ros2_control framework see `control.ros.org <https://control.ros.org/>`__.
+
+.. list-table::
+  :header-rows: 1
+
+  * - Package Name
+    - API
+    - ROS Index
+  * - controller_interface
+    - `API <http://docs.ros.org/en/rolling/p/controller_interface/>`__
+    - `ROS Index <https://index.ros.org/p/controller_interface/#rolling>`__
+  * - controller_manager
+    - `API <http://docs.ros.org/en/rolling/p/controller_manager/>`__
+    - `ROS Index <https://index.ros.org/p/controller_manager/#rolling>`__
+  * - controller_manager_msgs
+    - `API <http://docs.ros.org/en/rolling/p/controller_manager_msgs/>`__
+    - `ROS Index <https://index.ros.org/p/controller_manager_msgs/#rolling>`__
+  * - hardware_interface
+    - `API <http://docs.ros.org/en/rolling/p/hardware_interface/>`__
+    - `ROS Index <https://index.ros.org/p/hardware_interface/#rolling>`__
+  * - joint_limits
+    - `API <http://docs.ros.org/en/rolling/p/joint_limits/>`__
+    - `ROS Index <https://index.ros.org/p/joint_limits/#rolling>`__
+  * - ros2_control_test_assets
+    - `API <http://docs.ros.org/en/rolling/p/ros2_control_test_assets/>`__
+    - `ROS Index <https://index.ros.org/p/ros2_control_test_assets/#rolling>`__
+  * - transmission_interface
+    - `API <http://docs.ros.org/en/rolling/p/transmission_interface/>`__
```

Comment:
@christophfroehlich sorry for not writing earlier 

I see it's missing ros2controlcli, rqt_controller_manager etc

Reply:
is this really helpful? I'm not sure, this is not meant to be of public API?
http://docs.ros.org/en/rolling/p/ros2controlcli/
http://docs.ros.org/en/rolling/p/rqt_controller_manager/